src/pzt/postgresql/sections/10-try/30-test.mdx

```diff
@@ -0,0 +1,16 @@
+Now run the `psql` client with strict certificate verification, and check that SSL is in use:
+
+<CodeBlock autoCopy language="shell-session">
+  {`mkdir ~/.postgresql
+step ca root > ~/.postgresql/root.crt
+psql "postgresql://postgres@db.example.net:5432/postgres?sslmode=verify-full"`}
+</CodeBlock>
```

Comment:
When I dug in to postgres, it was actually really interesting/surprising to me that `psql` is basically insecure by default:

> By default, PostgreSQL will not perform any verification of the server certificate. This means that it is possible to spoof the server identity (for example by modifying a DNS record or by taking over the server IP address) without the client knowing. In order to prevent spoofing, the client must be able to verify the server's identity via a chain of trust. 

From that [doc you linked](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBQ-SSL-CERTIFICATES).

I think it might be worth calling it out more explicitly here (or in research notes) and how important `sslmode` is. It's crazy to me that it's not better by default, and potentially a very surprising fail-open for users.

Reply:
Great idea. Added.